### PR TITLE
nacos2.0.0 adapt

### PR DESCRIPTION
**Pull request by naah69**
// Describe your PR here; eg. Fixes #issueNo
issue #1188 

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by yu199195** [comment]
whats the modify?

**Comment by yu199195** [comment]
@naah69 I don't understand the meaning of your modification, Can you explain it to me

File: soul-admin/pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <artifactId>soul-admin</artifactId>
 
     <properties>
-        <nacos-client.version>1.2.0</nacos-client.version>
+        <nacos-client.version>2.0.0</nacos-client.version>
```
Comment (yu199195) [inline soul-admin/pom.xml:31]: maybe move to root pom.xml

Comment (naah69) [reply soul-admin/pom.xml:31]: it only change version of  nacos-client to provide performance of nacos.

commit c5bbe3c fix error when nacos username or password is null in config file.

**Pull request merged**